jira_to_google_calendar/program.py

```diff
@@ -16,39 +16,25 @@
 
 import autokitteh
 from autokitteh.google import google_calendar_client
-
-
-CALENDAR_CONNECTION_NAME = "my_google_calendar"
-JIRA_CONNECTION_NAME = "my_jira"
+from autokitteh.atlassian import get_url
 
 
 def on_jira_issue_created(event):
     """Workflow's entry-point, triggered by an incoming Jira event."""
-    details = _extract_issue_details(event.data.issue)
-    _create_calendar_event(details)
-
 
-def _extract_issue_details(issue):
-    base_url = os.getenv(JIRA_CONNECTION_NAME + "__BaseURL")
-    desc = f"Link to Jira Issue: {base_url}/browse/{issue.key}\n\n"
-    issue_details = autokitteh.AttrDict(
-        {
-            "description": desc + issue.fields.description,
-            "duedate": issue.fields.duedate,
-            "summary": issue.fields.summary,
-        }
-    )
-    return issue_details
+    _create_calendar_event(event.data.issue.fields, event.data.issue.key)
```

Comment:
Short and sweet, cool! Please just remove the empty line above.

Reply:
Done